pkg/kubelet/kubelet.go

```diff
@@ -3567,7 +3579,18 @@ func (kl *Kubelet) ListenAndServeReadOnly(address net.IP, port uint) {
 	server.ListenAndServeKubeletReadOnlyServer(kl, kl.resourceAnalyzer, address, port, kl.containerRuntime)
 }
 
-// Filter out events that are not worthy of pod syncing
+// deleteContainersInPod issues container deletion requests
+func (kl *Kubelet) deleteContainersInPod(candidates ContainerStatusbyCreated) {
+	for _, candidate := range candidates {
+		select {
+		case kl.containerDeletionWorker <- candidate.ID:
```

Comment:
Is this expected to be blocking? What is the interface here? Fire and continue or wait for success?